src/main/rules/GCI604/java/GCI604.asciidoc

```diff
@@ -0,0 +1,31 @@
+Spring Retry: Inconsistent configuration
+= Inconsistent Spring Retry Configuration
+
+An inconsistent configuration of Spring Retry can lead to blocking threads and increased energy usage.
+
+With Spring Retry, you can mark operations as retryable using `@Retryable`. However, misconfiguration (e.g., too many retries or long backoff periods) can block threads unnecessarily — consuming CPU cycles and energy.
+
+This rule applies to usages of `@Retryable` only.
+A threshold `max.timeout` defines the maximum acceptable blocking time (in milliseconds).
+
+`max.timeout=50000`
```

Comment:
what is the good value for max timeout ? maybe also describe de "maxDelay" parameter